modules/microshift-lvms-using.adoc

```diff
@@ -2,8 +2,8 @@
 //
 // * microshift_storage/microshift-storage-plugin-overview.adoc
 
-:_content-type: CONCEPT
-[id="using-lvms_{context}"]
+:_content-type: PROCEDURE
+[id="microshift-using-lvms_{context}"]
```

Comment:
I think the [anchor has to match the file name](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata) which looks like it was changed.